model/outgoing_webhook.go

```diff
@@ -199,7 +201,9 @@ func (o *OutgoingWebhook) PreUpdate() {
 }
 
 func (o *OutgoingWebhook) HasTriggerWord(word string) bool {
-	if len(o.TriggerWords) == 0 || len(word) == 0 {
+	if len(o.TriggerWords) == 0 {
+		return true
```

Comment:
Noted. Renamed the function to `TriggerWordExactMatch`. Also added another one as `GetTriggerWord `